vaadin-list-mixin.html

```diff
@@ -288,8 +288,15 @@
     focus() {
       // In initialisation (e.g vaadin-select) observer might not been run yet.
       this._observer && this._observer.flush();
-      const firstItem = this.querySelector('[tabindex="0"]') || (this.items ? this.items[0] : null);
-      firstItem && firstItem.focus();
+      const firstItem = this.querySelector('[tabindex="0"]');
+      if (firstItem) {
+        firstItem.focus();
```

Comment:
Should we consider calling `_focus` in this case too? This would scroll the focus element into the view.